docs/source/index.rst

```diff
@@ -113,12 +115,12 @@ that live in Africa:
 
 There are a couple of things to notice about queries:
 
-- All queries start with a vertex and branch to other vertices using **vertex fields**.
-- **Directives**, such as :code:`@output` and :code:`@filter`, specify the semantics of a query.
-  :code:`@output` indicates which properties to emit and :code:`@filter` specifies a filter
-  operation.
+- All queries start with a vertex and expand to other vertices using **vertex fields**.
+- **Directives** specify the semantics of a query. :code:`@output` indicates which properties to
+  emit. :code:`@filter` specifies a filter operation.
```

Comment:
`emit` is still here. How about `indicates the properties whose values should be returned`?

Reply:
Sorry I realized earlier that I had missed this comment and was about to submit a commit to address it. That sounds good.